packages/loader/container-definitions/src/deltas.ts

```diff
@@ -158,7 +158,11 @@ export interface IDeltaManager<T, U> extends IEventProvider<IDeltaManagerEvents>
 /** Events emitted by a Delta Queue */
 export interface IDeltaQueueEvents<T> extends IErrorEvent {
     (event: "push" | "op", listener: (task: T) => void);
-    (event: "idle", listener: () => void);
+    /**
+     * @param count - number of elements (T) processed before becoming idle
+     * @param duration - amount of time it took to process elements (milliseconds).
```

Comment:
BY elements you mean messages, right?

Reply:
IDeltaQueueEvents is really "generic" interface (it uses "events" terminology, so maybe it's better?)
But there is no mention of messages here, so I chose "elements" (though clarified that it's <T>).

Reply:
Changed to "events" to stay consistent with name of interface.